handlers/main.yml

```diff
@@ -21,7 +21,13 @@
     name: ufw
     state: restarted
 
-- name: Save iptables rules (Debian/Ubuntu and CentOS/RHEL/Fedora)
+- name: Enable iptables
+  ansible.builtin.systemd_service:
```

Comment:
Can we use instead name `ansible.builtin.systemd` here? because looks like name `ansible.builtin.systemd_service` available only in a brand new ansible version's. But as described in the docks - name `ansible.builtin.systemd` is kept as an alias for backward compatibility.

Reply:
Ok. changed to `ansible.builtin.systemd`.